src/views/Order.vue

```diff
@@ -1,118 +1,279 @@
+                </ion-label>
+                <ion-note slot="end" color="success">
+                  {{ $t("In Stock", { count: 15 }) }}
+                </ion-note>
+              </ion-item>
+              <ion-item>
+                <ion-label> {{ $t("Promise date") }} </ion-label>
+                <p slot="end">14 Jan 2021</p>
+              </ion-item>
+              <ion-item>
+                <ion-label> {{ $t("PO arrival date") }} </ion-label>
+                <p slot="end">14 Jan 2021</p>
+              </ion-item>
+              <ion-item>
+                <ion-label> {{ $t("Last brokered") }} </ion-label>
+                <p slot="end">California Warehouse</p>
+              </ion-item>
+            </ion-card>
+          </div>
+        </div>
       </div>
-      <div class="order-items">
-        <ion-card>
-          <ion-item>
-            <ion-thumbnail>
-              <Image/>
-            </ion-thumbnail>
-            <ion-label>
-              <h2>Brand</h2>
-              <h1>Virtual Name</h1>
-              <p>Color: color</p>
-              <p>Size: size</p>
-            </ion-label>
-          </ion-item>
-          <ion-item>
-            <ion-label>
-              Promise date
-            </ion-label>
-            <ion-text slot="end">
-              14 Jan 2021
-            </ion-text>
-          </ion-item>
-            <ion-item>
-            <ion-label>
-              Promise date
-            </ion-label>
-            <ion-text slot="end">
-              14 Jan 2021
-            </ion-text>
-          </ion-item>
-            <ion-item>
-            <ion-label>
-              Promise date
-            </ion-label>
-            <ion-text slot="end">
-              14 Jan 2021
-            </ion-text>
-          </ion-item>
-        </ion-card>
-      </div>
-    </div>
-    </ion-page>
+    </ion-content>
+  </ion-page>
 </template>
 
 <script>
-import Image from "../components/Image.vue"
+import Image from "../components/Image.vue";
+import {
+  IonBackButton,
+  IonBadge,
+  IonButton,
+  IonCard,
+  IonCardContent,
+  IonCheckbox,
+  IonChip,
+  IonContent,
+  IonHeader,
+  IonIcon,
+  IonItem,
+  IonLabel,
+  IonList,
+  IonListHeader,
+  IonNote,
+  IonPage,
+  IonSearchbar,
+  IonSelect,
+  IonThumbnail,
+  IonTitle,
+  IonToolbar,
+  IonSelectOption,
+} from "@ionic/vue";
+import { pricetag, ribbon } from "ionicons/icons";
 export default {
-    name:'Order',
-    components:{Image}
-}
+  name: "Order",
+  components: {
+    Image,
+    IonSelectOption,
+    IonBackButton,
+    IonBadge,
+    IonButton,
+    IonCard,
+    IonCardContent,
+    IonCheckbox,
+    IonChip,
+    IonContent,
+    IonHeader,
+    IonIcon,
+    IonItem,
+    IonLabel,
+    IonList,
+    IonListHeader,
+    IonNote,
+    IonPage,
+    IonSearchbar,
+    IonSelect,
+    IonThumbnail,
+    IonTitle,
+    IonToolbar,
+  },
+  setup() {
+    return {
+      pricetag,
+      ribbon,
+    };
+  },
+};
 </script>
 <style scoped>
```

Comment:
Improve indentation and place styling in order in which they are used.

Reply:
done